01week/resume/index.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <title>Page Title</title>
+    <link rel="stylesheet" href="./css/style.css">
+  </head>
+
+  <body>
+
+    <header>
+      <h1>lisa umhey</h1>
+      <p>lisa.umhey@gmail.com | 818.519.7081 | austin, tx 78702</p>
+    </header>
+
+    <nav></nav>
+
+    <section>
+      <h1>summary of qualifications</h1>
+        <ul>
+          <li>Double B.A., M.S.Ed, and 9 years of professional experience</li>
+          <li>Highly reflective and actively seeks opportunities to learn and grow</li>
+          <li>Naturally builds positive, productive, and caring relationships with colleagues and clients</li>
+          <li>Intrinsically motivated to create, solve problems and succeed</li>
+        </ul>
+    </section>
+
+    <section>
+        <h1>professional experience</h1>
+          <h4>Florist. Rosehip Flora. Austin, TX. 2017 - Present.</h4>
+            <ul>
+              <li>Collaborates with venues, vendors, colleagues and clients to produce custom foral design for weddings, events, and local businesses</li>
+              <li>Demonstrates agility, self-direction, professionalism, meticulousness, and vision to create under high-pressure/high-stakes conditions</li>
+              <li>Communicates with precision about technical processes</li>
+            </ul>
+          <h4>Recruitment Manager. KIPP Austin Public Schools. Austin, TX. 2013-2016.</h4>
+            <ul>
+              <li>Performed outreach, fielded applications, interviewed candidates, and communicated
+hiring recommendations to fill roughly 85 full-time positions across 4 elementary schools
```

Comment:
I would have made this text line up with the line above it. This would make it easier to read, and wont matter for the browser since the browser will collapse all white space to a single space.